lib/miq_memcached.rb

```diff
@@ -7,7 +7,10 @@ def self.server_address
 
   def self.default_client_options
     options = {
-      :pool_size => 1,
+      # Direct Dalli Clients won't use connection pool but will be threadsafe.
+      # ActiveSupport::Cache::MemCacheStore and ManageIQ::Session::MemCacheStoreAdapter
+      # use threadsafe but also accept connection pool options.
+      :threadsafe => true
```

Comment:
pool_size was ignored for regular Dalli Clients using MiqMemcached, but was inherited for the other two clients below, causing connection errors when the 1 sized pool was checked out.